spread/runner.go

```diff
@@ -36,6 +36,8 @@ type Options struct {
 	Seed           int64
 	Repeat         int
 	GarbageCollect bool
+	XUnit          bool
+	JSONUnit          bool
```

Comment:
indentation seems off here, did you run gofmt?

Reply:
added, thanks